### PR TITLE
Demo video

### PR DESCRIPTION
**Pull request by wnmprk**

### CODE REVIEW

File: browser/js/common/directives/shelf/shelf.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <div ng-repeat='category in swags' class='row'>
 	<h1>{{category[0].category | uppercase }}</h1>
 	<swag ng-repeat='swag in category'></swag>
-	<hr class='shelf'>
```
Comment (rpastor90) [inline browser/js/common/directives/shelf/shelf.html:4]: what breaks up the sections now? is it just the names of the categories?

File: browser/js/firstTimeUser/firstTimeUser.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ app.config(function($stateProvider) {
             authenticate: true
         },
         resolve: {
-            animals: function(AnimalFactory) {
-                return AnimalFactory.fetchAnimals();
+            panda: function(PandaFactory) {
+                return PandaFactory.getPanda();
```
Comment (rpastor90) [inline browser/js/firstTimeUser/firstTimeUser.js]: This isn't async, so I think we don't need to have it in a resolve block

File: browser/js/firstTimeUser/firstTimeUser.js [hunk 3]
```diff
@@ -17,25 +17,15 @@ app.config(function($stateProvider) {
     });
 });
 
-app.controller('FirstTimeUserCtrl', function($scope, $state, UserFactory, animals, user, AuthService) {
+app.controller('FirstTimeUserCtrl', function($scope, $state, UserFactory, panda, user) {
     $scope.user = user;
-
-    $scope.update = function(user) {
-        return UserFactory.updateUser(user)
+    $scope.panda = panda;
```
Comment (rpastor90) [inline browser/js/firstTimeUser/firstTimeUser.js]: scope.panda can just be equal to `PandaFactory.getPanda()`

File: server/app/routes/users.js [hunk 4]
```diff
@@ -145,7 +145,7 @@ router.put('/:userId/addFriend', ensureAuthenticated, function(req, res, next) {
 
 router.put('/:userId/addPost', ensureAuthenticated, function(req, res, next) {
     User.findOne({ _id: req.params.userId })
-    .then(function(user){
+    .then(function(){
         return User.findOne({ _id: req.body.friend._id})
```
Comment (rpastor90) [inline server/app/routes/users.js:149]: just note this, looks like we're doing two queries when we probably only need the second one??? Not too sure

File: browser/js/common/directives/shelf/shelf.html (hunk 1 above)
Comment (wnmprk) [inline browser/js/common/directives/shelf/shelf.html:4]: yeah, just thought aesthetically looked better without the tie-dye shelves lol

File: browser/js/firstTimeUser/firstTimeUser.js (hunk 2 above)
Comment (wnmprk) [inline browser/js/firstTimeUser/firstTimeUser.js]: kay removed panda

File: server/app/routes/users.js (hunk 4 above)
Comment (wnmprk) [inline server/app/routes/users.js:149]: i removed user from the parameter cause it was a lint error. not sure whats happening here either...

**Pull request merged**